guides/common/modules/proc_tracking-subscription-usage-by-using-the-subscriptions-service.adoc

```diff
@@ -0,0 +1,39 @@
+[id="tracking-subscription-usage-by-using-the-subscriptions-service"]
+= Tracking subscription usage by using the Subscriptions service
+
+You can configure your {ProjectServer} to report usage data to the {RHCloud} by enabling and adjusting the `foreman_rh_cloud` plugin.
```

Comment:
```suggestion
You can configure your {ProjectServer} to report usage data to the {RHCloud} by using the `foreman_rh_cloud` plugin.
```
I'd go for a more general verb because you don't have to enable it in Satellite.